src/classes/MixPlaylist.ts

```diff
@@ -0,0 +1,76 @@
+import { Base, BaseAttributes, ChannelCompact, VideoCompact } from ".";
+import { YoutubeRawData } from "../common";
+
+/** @hidden */
+interface PlaylistAttributes extends BaseAttributes {
+    title: string;
+    videoCount: number;
+    viewCount?: number | null;
```

Comment:
Please remove the unused attributes listed here: https://github.com/SuspiciousLookingOwl/youtubei/pull/29#issuecomment-921427635